apps/condo/schema.graphql

```diff
@@ -59394,6 +59394,12 @@ type ExportMeterReadingsOutput {
   linkToFile: String!
 }
 
+input _internalDeleteMeterAndMeterReadingsInput {
+  dv: Int!
+  sender: JSON!
+  propertyIds: [String]!
+}
```

Comment:
you need to use maketypes to change it to SenderField, (but in our schema.graphql we have lots of sender with type JSON and some sender with type SenderField)